src/utils/hardCodedSizes/index.js

```diff
@@ -0,0 +1,3 @@
+export const mobileMediaQuerySize = 640
+
+export const navigationWidth = 200
```

Comment:
Some hard-coded sizes needed as tachyons has too large of jumps between higher relative numbers (4-9).